docs/reference/python/katana.local.graph.rst

```diff
@@ -7,7 +7,3 @@ A ``Graph`` is created by calling one of the import routines from
 :py:mod:`~katana.local.import_data`.
 
 .. autoclass:: katana.local.Graph
-   :members:
-   :inherited-members:
-   :undoc-members:
-   :special-members: __init__, __iter__, __getitem__, __setitem__, __len__
```

Comment:
Should `__iter__` and `__len__` be removed from the current documentation? There exists `Graph.num_nodes` for getting the number of nodes. How should users iterate through the nodes?

Reply:
I shouldn't have removed them here. We may be changing this API a bit, but for the moment, `__len__` and `__iter__` should be documented on `Graph` as you say. I will add this back.